github-actions/oss-readme-template/src/main.ts

```diff
@@ -68,9 +61,14 @@ function getProjectInfo(projectType: ProjectType): ProjectInfo {
       required: true,
       trimWhitespace: true,
     });
+    const usageExamplePath = core.getInput('usage_example_path', {
+      required: false,
```

Comment:
here, since we are only running this for SDKs, you can say required: `true`.  That way the build will fail if people omit it for our SDK repos, which will ensure that we always have a usage example for each SDK repo.